src/cmd-run

```diff
@@ -184,13 +172,38 @@ EOF
         exit 1
     fi
 fi
-set -- -fw_cfg name=opt/com.coreos/config,file="${IGNITION_CONFIG_FILE}" "$@"
+
+if [ -z "${VM_PERSIST_IMG}" ]; then
+    echo "Creating temporary image"
+    VM_IMG=$(mktemp -p "${TMPDIR:-/var/tmp}")
+    qemu-img create -q -f qcow2 -b "${VM_DISK}" "${VM_IMG}"
+else
+    echo "Re-using existing ${VM_PERSIST_IMG}"
+    VM_IMG=${VM_PERSIST_IMG}
+fi
+
+if [ "$(arch)" == "ppc64le" ] || [ "$(arch)" == "s390x" ]; then
+    echo "fw_cfg is not supported, injecting config.ign via guestfish"
+    coreos_gf_run_mount "${VM_IMG}"
+    coreos_gf mkdir-p /boot/ignition
+    coreos_gf upload ${IGNITION_CONFIG_FILE} /boot/ignition/config.ign
+    #TODO coreos_gf_relabel /boot/ignition/config.ign
```

Comment:
Interesting that we don't need to relabel.  I guess because we load this before SELinux policy is loaded?